src/System.Management.Automation/security/MshSignature.cs

```diff
@@ -389,5 +400,31 @@ private static string GetSignatureStatusMessage(SignatureStatus status,
 
             return message;
         }
+
+        /// <summary>
+        /// Extracts the Subject Alternative Name from the certificate.
+        /// </summary>
+        /// <param name="certificate">The certificate to extract SAN from.</param>
+        /// <returns>Formatted SAN string or null if not found.</returns>
+        private static string GetSubjectAlternativeName(X509Certificate2 certificate)
+        {
+            if (certificate == null)
+            {
+                return null;
+            }
+
+            // Look for the Subject Alternative Name extension (OID: 2.5.29.17)
+            foreach (X509Extension extension in certificate.Extensions)
+            {
+                // Check if this is the Subject Alternative Name extension
+                if (extension.Oid != null && extension.Oid.Value == "2.5.29.17")
+                {
+                    // Format the extension with multiline format (parameter 1)
+                    return extension.Format(multiLine: true);
```

Comment:
Interesting, why is it multiline with prefixes?
If there could be some values I'd expect the type should be `string[]`.

Reply:
@iSazonov I think I had it resolve this.

Reply:
https://github.com/PowerShell/PowerShell/pull/26252#pullrequestreview-3362503063